src/ffs_postgres/20-ffs_data.sql

```diff
@@ -0,0 +1,15 @@
+-- Copyright The OpenTelemetry Authors
+-- SPDX-License-Identifier: Apache-2.0
+
+-- Feature Flags created and initialized on startup
+--     'enabled' is a decimal value between 0 and 1 (inclusive)
+--     0.0 is always disabled
+--     1.0 is always enabled
+--     All values between set a percentage chance on each request
+--     example: 0.55 is enabled 55% of the time
+INSERT INTO public.featureflags (name, description, enabled)
+VALUES
+    ('productCatalogFailure', 'Fail product catalog service on a specific product', 0),
+    ('recommendationCache', 'Cache recommendations', 0),
+    ('adServiceFailure', 'Fail ad service requests sporadically', 0),
```

Comment:
As this is not random anymore, I think we can get rid of the `sporadically`.

```suggestion
    ('adServiceFailure', 'Fail ad service requests', 0),
```